mythril/laser/smt/bitvecfunc.py

```diff
@@ -26,18 +26,19 @@ def _arithmetic_helper(
     union = a.annotations.union(b.annotations)
 
     if isinstance(b, BitVecFunc):
-        # TODO: Find better value to set input and name to in this case?
-        input_string = "MisleadingNotationop(invhash({}) {} invhash({})".format(
-            hash(a), operation, hash(b)
-        )
         return BitVecFunc(
             raw=raw,
             func_name="Hybrid",
-            input_=BitVec(z3.BitVec(input_string, 256), annotations=union),
+            input_=BitVec(z3.BitVec("", 256), annotations=union),
+            nested_functions=a.nested_functions + b.nested_functions + [a, b],
```

Comment:
No inverse of functions exist for our current BitVecFunc supporting functions, So hash(a) + hash(b)'s inverse has no use(or no one can find it). We can return a BitVec in these cases right?

Reply:
But you want to be able to compare `hash(a) +hash(b) == hash(c) + hash(d)`?

Reply:
For that we shouldn't directly add the two nested functions lists right? We might have problems with commutative stuff.

Reply:
Could you give an example?

Reply:
Oh! you are using product() nevermind.